plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -0,0 +1,773 @@
+			[
+				'id'        => 'product-inventory-email',
+				'blockName' => 'woocommerce/product-inventory-email-field',
+			]
+		);
+
+		$product_inventory_advanced_wrapper->add_block(
+			[
+				'id'         => 'product-limit-purchase',
+				'blockName'  => 'woocommerce/product-checkbox-field',
+				'attributes' => [
+					'title'    => __(
+						'Restrictions',
+						'woocommerce'
+					),
+					'label'    => __(
+						'Limit purchases to 1 item per order',
+						'woocommerce'
+					),
+					'property' => 'sold_individually',
+					'tooltip'  => __(
+						'When checked, customers will be able to purchase only 1 item in a single order. This is particularly useful for items that have limited quantity, like art or handmade goods.',
+						'woocommerce'
+					),
+				],
+			]
+		);
+	}
+
+	/**
+	 * Adds the shipping group blocks to the template.
+	 */
+	private function add_shipping_group_blocks() {
+		$shipping_group = $this->add_group(
+			[
+				'id'         => 'shipping',
+				'order'      => 40,
+				'attributes' => [
+					'id'    => 'shipping',
+					'title' => __( 'Shipping', 'woocommerce' ),
+				],
+			]
+		);
+		$shipping_group->add_block(
+			[
+				'id'         => 'product_variation_notice_shipping_tab',
+				'blockName'  => 'woocommerce/product-has-variations-notice',
+				'attributes' => [
+					'id'         => 'wc-product-notice-has-options',
+					'content'    => __( 'This product has options, such as size or color. You can now manage each variation\'s price and other details individually.', 'woocommerce' ),
+					'buttonText' => __( 'Go to Variations', 'woocommerce' ),
+					'type'       => 'info',
+				],
+			]
+		);
+		// Product Pricing Section.
+		$product_fee_and_dimensions_section = $shipping_group->add_section(
+			[
+				'id'         => 'product-fee-and-dimensions-section',
+				'attributes' => [
+					'title'       => __( 'Fees & dimensions', 'woocommerce' ),
+					'description' => sprintf(
+					/* translators: %1$s: How to get started? link opening tag. %2$s: How to get started? link closing tag.*/
+						__( 'Set up shipping costs and enter dimensions used for accurate rate calculations. %1$sHow to get started?%2$s.', 'woocommerce' ),
+						'<a href="https://woocommerce.com/posts/how-to-calculate-shipping-costs-for-your-woocommerce-store/" target="_blank" rel="noreferrer">',
+						'</a>'
+					),
+				],
+			]
+		);
+		$product_fee_and_dimensions_section->add_block(
+			[
+				'id'        => 'product-shipping-class',
+				'blockName' => 'woocommerce/product-shipping-class-field',
+			]
+		);
+		$product_fee_and_dimensions_section->add_block(
+			[
+				'id'        => 'product-shipping-dimensions',
+				'blockName' => 'woocommerce/product-shipping-dimensions-fields',
+			]
+		);
+	}
+
+	/**
+	 * Adds the variation group blocks to the template.
+	 */
+	private function add_variation_group_blocks() {
+		$variation_group  = $this->add_group(
+			[
+				'id'         => 'variations',
+				'order'      => 50,
```

Comment:
Just thinking... all these `order` for the groups are specified all over the place in this file. What is all the `add_group` calls were done in one place and then blocks were added to the groups in these separate methods?

Reply:
Moved to a separate method as you suggested.